src/sweetalert2.js

```diff
@@ -640,7 +640,18 @@
 
     // Content
     if (params.text || params.html) {
-      $content.innerHTML = params.html || ('<p>' + escapeHtml(params.text.split('\n').join('<br>')) + '</p>');
+      if (typeof params.html === 'object') {
+        $content.innerHTML = '';
+        if (0 in params.html) {
```

Comment:
no need actually, I got it - nice trick! btw Array.isArray doesn't work as expected here